src/main/java/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -14,5 +22,13 @@ public DefaultOperator(String name) {
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		RobotMap.HumanInput.Operator.joystick.button3.onlyWhileHeld(new HatchIntake());
```

Comment:
No option to outtake? If we have a spare button we should have this so the drivers can try to unjam it if it gets jammed